jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/TravelerMessageFromHumanToAsnConverter.java

```diff
@@ -38,1082 +38,1258 @@
-   private static final String CENTER = "center";
-   private static final String NODE_LAT = "nodeLat";
-   private static final String NODE_LONG = "nodeLong";
-   private static final String Y = "y";
-   private static final String X = "x";
-   private static final String NODE_LAT_LON = "node-LatLon";
-   private static final String LON = "lon";
-   private static final String LAT = "lat";
-   private static final String NODE_XY = "node-XY";
-   private static final String NODE_XY2 = "NodeXY";
-   private static final String COMPUTED = "computed";
-   private static final String SPEED_LIMITS = "speedLimits";
-   private static final String LANE_ANGLE = "laneAngle";
-   private static final String LANE_CROWN_POINT_RIGHT = "laneCrownPointRight";
-   private static final String LANE_CROWN_POINT_LEFT = "laneCrownPointLeft";
-   private static final String LANE_CROWN_POINT_CENTER = "laneCrownPointCenter";
-   private static final String D_ELEVATION = "dElevation";
-   private static final String D_WIDTH = "dWidth";
-   private static final String DATA = "data";
-   private static final String OFFSET_X_AXIS = "offsetXaxis";
-   private static final String OFFSET_Y_AXIS = "offsetYaxis";
-   private static final String ROTATE_XY = "rotateXY";
-   private static final String SCALE_X_AXIS = "scaleXaxis";
-   private static final String SCALE_Y_AXIS = "scaleYaxis";
-   private static final String NODE_LIST = "nodeList";
-   private static final String REGION_POINT_SET = "regionPointSet";
-   private static final String CIRCLE = "circle";
-   private static final String SHAPE_POINT_SET = "shapePointSet";
-   private static final String DELTA = "delta";
-   private static final String OFFSET = "offset";
-   private static final String NODES = "nodes";
-   private static final String XY = "xy";
-   private static final String LL = "ll";
-   private static final String OLD_REGION = "oldRegion";
-   private static final String GEOMETRY = "geometry";
-   private static final String PATH = "path";
-   private static final String DESCRIPTION = "description";
-   private static final String CLOSED_PATH = "closedPath";
-   private static final String ANCHOR = "anchor";
-   private static final String ID = "id";
-   private static final String REGION = "region";
-   private static final String LANE_WIDTH = "laneWidth";
-   private static final String ANCHOR_POSITION = "anchorPosition";
-   private static final String REGULATOR_ID = "regulatorID";
-   private static final String SEGMENT_ID = "segmentID";
-   private static final String POSITION = "position";
-   private static final String TEXT = "text";
-   private static final String ITIS = "itis";
-   private static final String ITEM = "item";
-   private static final String START_DATE_TIME = "startDateTime";
-   private static final String DURATON_TIME_MISSPELLED = "duratonTime"; // J2735 2016 Misspelling 
-   private static final String DURATION_TIME = "durationTime"; // used in J2735 2020

```

Comment:
Did they remove duration time?

Reply:
The `durationTime` field was not removed. However, since we are no longer replacing the misspelled `duratonTime` field with `durationTime` in this class, this static String is no longer needed.